ansible/inventory/env/group_vars/all.yml

```diff
@@ -58,6 +58,8 @@ keycloak_postgres_password: "{{core_vault_postgres_password}}"
 kong_postgres_user: "{{core_vault_postgres_username}}"
 kong_postgres_password: "{{core_vault_postgres_password}}"
 enc_postgres_user: "{{core_vault_postgres_username}}"
+uci_postgres_user: "{{core_vault_uci_username}}"
+uci_postgres_password: "{{core_vault_uci_password}}"
```

Comment:
Can you change the value to {{ core_vault_postgres_password }}" , we can override this variable if we want different password.